tests/test_cli.sh

```diff
@@ -105,19 +106,9 @@ if [ -e "/var/tmp/test-results" ]; then
     rm -rf "/var/tmp/test-results"
 fi
 
-export BEAKERLIB_JOURNAL=0
-if [ -z "$*" ]; then
-    # invoke cli/ tests which can be executed without special preparation
-    ./tests/cli/test_blueprints_sanity.sh
-    ./tests/cli/test_compose_sanity.sh
-else
-    # execute other cli tests which need more adjustments in the calling environment
-    # or can't be executed inside Travis CI
-    for TEST in "$@"; do
-        $TEST
-    done
-fi
+setup_beakerlib_env
 
+run_beakerlib_tests "$@"
```

Comment:
@bcl with this change `make test_cli` changes behavior and that will affect how you run things locally. Do you want to have the previous if-else block back ?

Reply:
No, when I'm running it locally I pass in the list of tests via the RUN_TESTS variable in the makefile so it will be set.